### PR TITLE
[2] Changes to the bugreportutils.go file

### PR DESCRIPTION
**Pull request by chenj47**
Main change to this file:

1. Re-format the lines to enforce 80-character limit.
2. RequestMode used to have type string and held values like "continuous", "on-change", etc.
  Change it to be int32.
3. Instead of getting Min/Max Delay values for each sensor, use Max/Min Rate (in Hz) instead.

Changes added upon pull request @ #22

### CODE REVIEW

File: bugreportutils/bugreportutils.go [hunk 1]
```diff
@@ -41,80 +41,93 @@ const (
...
-	modelNameRE = regexp.MustCompile(`\[ro.product.model\]:\s+\[(?P<modelName>.*)\]`)
+	// modelNameRE is a regular expression that finds the model name line
+	// in the System Properties section of a bug report.
+	modelNameRE = regexp.MustCompile(`\[ro.product.model\]:\s+` +
+		`\[(?P<modelName>.*)\]`)
 
 	// pidRE is a regular expression to match PID to app name and UID.
-	pidRE = regexp.MustCompile(`PID #` + `(?P<pid>\d+)` + `: ProcessRecord[^:]+:` +
-		`(?P<app>[^/]+)` + `/` + `(?P<uid>.*)` + `}`)
+	pidRE = regexp.MustCompile(`PID #` + `(?P<pid>\d+)` +
```
Comment (agarwal-v) [inline bugreportutils/bugreportutils.go:67]: This seems like an unnecessary change. Can you double check the length of the line.

File: bugreportutils/bugreportutils.go [hunk 2]
```diff
@@ -41,80 +41,93 @@ const (
...
-	// sensorLineTwoRE is a regular expression to match the second line of sensor information
-	// from the sensor list line in the sensorservice dump in the bugreport starting
-	// from NRD42 and onwards.
-	sensorLineTwoRE = regexp.MustCompile(`\s*(?P<requestMode>[^|]+)` + `\s*\|` +
-		`(?P<variableOne>[^|]+)` + `\s*\|` + `(?P<variableTwo>[^|]+)` + `\s*\|` +
-		`(?P<batching>[^|]+)` + `\s*\|` + `(?P<wakeUp>[^|]+)`)
-
-	// BatchingDataRE is a regular expression that matches the max and reserved data quantity
-	// from the sensor infomration listed in the sensorservice dump of a bug report.
+		`(?P<sensorManufacturer>[^|]+)` + `\|` + `(\s*version=` +
```
Comment (agarwal-v) [inline bugreportutils/bugreportutils.go]: nit: I'd put version=<versionNumber> on the same line to increase readability.

File: bugreportutils/bugreportutils.go [hunk 3]
```diff
@@ -525,8 +561,10 @@ func TimeStampToMs(timestamp, remainder string, loc *time.Location) (int64, erro
 // SecFractionAsMs converts the fraction of a second to milliseconds.
 // e.g. "123456" from "27.123456" corresponds to 123ms (and 27 seconds).
 func SecFractionAsMs(fr string) (int64, error) {
-	// The string will be parsed as ms, so only the leading 3 digits of the string are used.
-	// Make sure the remainder has at least 3 digits, so the slice operation doesn't fail.
+	// The string will be parsed as ms, so only the leading 3 digits of the
+	// string are used.
```
Comment (agarwal-v) [inline bugreportutils/bugreportutils.go]: No need to start the next comment on a separate line.

**Review by agarwal-v — COMMENTED** [review]
Few nits

File: bugreportutils/bugreportutils.go (hunk 1 above)
Comment (chenj47) [reply bugreportutils/bugreportutils.go:67]: Locally I have tab size = 4, the old line exceeds the 80-character limit by just 1 character, so I made this change.

Comment (agarwal-v) [reply bugreportutils/bugreportutils.go:67]: Default at google is using 2. If you want you can skip this change

**Review by agarwal-v — APPROVED** [review]
